lib/pa11y.js

```diff
@@ -305,27 +319,33 @@ async function injectRunners(options, state) {
  * Sends a request to the page to instruct the injected pa11y script to run with the
  *   provided options
  * @private
- * @param {Object} options - Options to change the way tests run.
- * @param {Object} state - The current pa11y internal state, fields will be mutated by
+ * @param {Pa11yConfiguration} options - Options to change the way tests run.
+ * @param {object} state - The current pa11y internal state, fields will be mutated by
  *   this function.
- * @returns {Promise} A promise which resolves with the results of the pa11y evaluation
+ * @returns {Promise} A promise of the results of Pa11y's evaluation
  */
-function runPa11yWithOptions(options, state) {
-	/* eslint-disable no-underscore-dangle */
-	return state.page.evaluate(runOptions => {
-		return window.__pa11y.run(runOptions);
-	}, {
-		hideElements: options.hideElements,
-		ignore: options.ignore,
-		pa11yVersion: pkg.version,
-		rootElement: options.rootElement,
-		rules: options.rules,
-		runners: options.runners,
-		standard: options.standard,
-		wait: options.wait
-	});
-	/* eslint-enable no-underscore-dangle */
-}
+const runPa11yWithOptions = ({
+	hideElements,
+	ignore,
+	rootElement,
+	rules,
+	runners,
+	standard,
+	wait
+}, state) => state.page.evaluate(
+	/* eslint-disable-next-line no-underscore-dangle */
+	runOptions => window.__pa11y.run(runOptions),
+	{
+		pa11yVersion,
+		hideElements,
+		ignore,
+		rootElement,
+		rules,
+		runners,
+		standard,
+		wait
+	}
+);
```

Comment:
It looks like `` is not being passed to the runner:

```suggestion
const runPa11yWithOptions = ({
	hideElements,
	ignore,
	levelCapForReviewRequired,
	rootElement,
	rules,
	runners,
	standard,
	wait
}, state) => state.page.evaluate(
	/* eslint-disable-next-line no-underscore-dangle */
	runOptions => window.__pa11y.run(runOptions),
	{
		pa11yVersion,
		hideElements,
		ignore,
		levelCapForReviewRequired,
		rootElement,
		rules,
		runners,
		standard,
		wait
	}
);
```

With this, the JSON results looked correct, but the CLI reporter was showing `undefined` for the severity and I didn't have a chance to fully sort that out.

Reply:
Yikes, thanks @aarongoldenthal.

Reply:
Hi again @aarongoldenthal 

1. We could add a [set of choices](https://www.npmjs.com/package/commander#more-configuration) to the input as validation, but this Commander interface (`addOption`) isn't one we're using yet, so ~I'd prefer to do that as part of a pass over the whole Commander interface in case it has changed in the meantime. We're on version 11 and they're up to version 14 (memory fail - we're on 14 too)~ I'll have a look.
2. My ideas:
    - `level-cap-[if, when]-needs-review`
    - `level-cap-[if, when]-incomplete`
    - 🤷🏽‍♂️

Reply:
Hi @danyalaytekin 
1. I could see going either way. I don't believe we do the same for other inputs, for example `level`, so maybe a separate change for broader input validation.
2. I prefer `when` versus `if`, and I think `needs-review` gives an explicit action, so I'd vote for `level-cap-when-needs-review`